master/pkg/stream/stream.go

```diff
@@ -109,15 +147,23 @@ func (p *Publisher[T]) Broadcast(events []Event[T]) {
 
 	// check each event against each subscription
 	for _, sub := range p.Subscriptions {
-		func(){
+		func() {
 			for _, ev := range events {
-				var msg *websocket.PreparedMessage
+				var msg interface{}
 				if ev.After != nil && sub.filter(*ev.After) {
+					if ev.upsertCache == nil {
+						ev.upsertCache = sub.Streamer.PrepareFn(ev.After)
+					}
 					// update, insert, or fallin: send the record to the client.
```

Comment:
this comment is describing what the significance of this `if` block is.  It belongs at the top of the block, above the new `if ev.upsertCache == nil {` line.